ROADMAP.md

```diff
@@ -0,0 +1,9 @@
+# Lima roadmap
+
+Instead of using a static text file, Lima uses the `roadmap` label on Github issues to designate features or bug fixes that we plan to implement.
+
+Issues are tagged with the `roadmap` label when at least one maintainer or contributor has declared intent to work on or help with the implementation.
+
+Since all development is done by volunteers, there are no commitments or timelines attached to the label, and the label may be removed again from an abandoned issue at any time.
```

Comment:
"all development is done by volunteers" might be misleading as some of us are paid by the employers to work on Lima

https://en.wiktionary.org/wiki/volunteer
> One who enters into, or offers for, any service of their own free will, especially when done without pay.

Maybe just remove "Since all development is done by volunteers, "?